lang/fr-on.js

```diff
@@ -13,6 +13,6 @@ export default {
 	"intl-common:characters:questionMark": "point d'interrogation", // short name or description of the "?" character
 	"intl-common:characters:quotationMark": "guillemet", // short name or description of the '"' character
 	"intl-common:navigation:back:title": "Retour", // term for navigating back to the previous page (in Title Case)
-	"intl-common:navigation:next:title": "Suivant", // term for navigating to the next page or item (in Title Case)
+	"intl-common:navigation:next:title": "Page suivante", // term for navigating to the next page or item (in Title Case)
 	"intl-common:navigation:previous:title": "Précédent", // term for navigating to the previous page or item (in Title Case)
```

Comment:
I think we definitely don't want "Page" here for 2 reasons:
1. The `previous:title` term value doesn't have it
2. This isn't always about navigating to the next/previous "page", it could be iterating between users, assignments, etc.

Reply:
Agreed